pkg/envoy/cds/cluster_test.go

```diff
@@ -63,6 +63,20 @@ func TestGetUpstreamServiceCluster(t *testing.T) {
 			assert.Equal(tc.expectedLbPolicy, remoteCluster.LbPolicy)
 		})
 	}
+
+	t.Run("adds health checks when configured", func(t *testing.T) {
+		assert := tassert.New(t)
+		remoteCluster, err := getUpstreamServiceCluster(downstreamSvcAccount, upstreamSvc, mockConfigurator, withActiveHealthChecks)
+		assert.NoError(err)
+		assert.NotNil(remoteCluster.HealthChecks)
+	})
+
+	t.Run("does not add health checks when not configured", func(t *testing.T) {
+		assert := tassert.New(t)
+		remoteCluster, err := getUpstreamServiceCluster(downstreamSvcAccount, upstreamSvc, mockConfigurator)
+		assert.NoError(err)
+		assert.Nil(remoteCluster.HealthChecks)
+	})
```

Comment:
Could you augment the table of tests with the healthCheck option for consistency

Reply:
I made these separate because the table tests assert on parts of the config that aren't relevant to health checking. I suppose I could have each test case define a function that does its own asserts. Did you have something else in mind?

Reply:
In the future, I'd like to be able to set the cluster options directly in the test case, and test a subset of cluster fields that are currently being tested. It would require tweaking the existing test which I wouldn't suggest holding this patch on.